### DISCUSSION TITLE
Argument for forcing Dark Mode

### DISCUSSION THREAD

**Discussion by kriiv** [spatie/browsershot#851] [General]
I'm trying to force dark mode when rendering HTML and screenshotting.

Code is as follows however still not seeing dark mode at all:

```
Browsershot::html($html_content)
                ->fullPage()
                ->disableJavascript()
                ->addChromiumArguments(['--force-dark-mode', '--enable-features=WebContentsForceDark'])
                ->save($screenshot_path);
```

Any ideas for arguments that do work?

**Comment by AlexVanderbist** [comment]
Hey, we'll tag this in a minute: https://github.com/spatie/browsershot/pull/890
